src/constants.ts

```diff
@@ -0,0 +1,25 @@
+export const servicePathPrefix = "/presenter-account";
+
+export enum URL_QUERY_PARAM {
```

Comment:
Do we use query params at any point in the journey?

Reply:
No. I just copied as part of. arefactor I'll remove.